### PR TITLE
nat orb

### PR DESCRIPTION
**Pull request by y-yao**

### CODE REVIEW

File: src/chem/chem_system.cc [hunk 1]
```diff
@@ -511,3 +531,554 @@ double ChemSystem::get_s2() const {
   }
   return s2;
 }
+
+//=====================================================
+MatrixXd ChemSystem::get_1rdm() const {
```
Comment (jl2922) [inline src/chem/chem_system.cc]: Seems really long, better to put into a separate module.

File: src/chem/chem_system.cc [hunk 2]
```diff
@@ -511,3 +531,554 @@ double ChemSystem::get_s2() const {
...
+    for (unsigned i=0; i<n; i++) {
+      eigenvalues[inds[irrep][i]] = tmp_eigenvalues(n-i-1);
+      for (unsigned j=0; j<n; j++) {
+        rot(inds[irrep][i],inds[irrep][j]) = tmp_eigenvectors(i,n-j-1); 
+      }
+    } 
+  } // irrep
+
+  std::cout<< "Occupation numbers:\n";
+  for (unsigned i=0; i<n_orbs;i++) std::cout<< eigenvalues[i] <<"\n";
```
Comment (jl2922) [inline src/chem/chem_system.cc]: Could you format them as the other parts of the code? The format is specified by https://github.com/jl2922/shci/blob/master/.clang-format and works with all major code editors. On vim, I use https://github.com/google/vim-codefmt

File: src/chem/chem_system.cc (hunk 1 above)
Comment (y-yao) [reply src/chem/chem_system.cc]: Done

File: src/chem/integrals.cc [hunk 3]
```diff
@@ -218,12 +218,14 @@ std::vector<double> Integrals::get_orb_energies() const {
 }
 
 void Integrals::reorder_orbs(const std::vector<double>& orb_energies) {
-  std::vector<unsigned> orb_order(n_orbs);
+  //std::vector<unsigned> orb_order(n_orbs);
```
[the remark this answers is not in the document]
Comment (y-yao) [reply src/chem/integrals.cc]: Done

File: src/chem/integrals.h [hunk 4]
```diff
@@ -63,12 +67,12 @@ class Integrals {
...
 void Integrals::serialize(B& buf) const {
-  buf << energy_core << n_orbs << n_elecs << n_up << n_dn << orb_sym << det_hf;
+  buf << energy_core << n_orbs << n_elecs << n_up << n_dn << orb_sym << orb_order << orb_order_inv << det_hf;
   buf << integrals_1b << integrals_2b;
 }
 
 template <class B>
 void Integrals::parse(B& buf) {
-  buf >> energy_core >> n_orbs >> n_elecs >> n_up >> n_dn >> orb_sym >> det_hf;
+  buf >> energy_core >> n_orbs >> n_elecs >> n_up >> n_dn >> orb_sym >> orb_order >> orb_order_inv >> det_hf;
```
[the remark this answers is not in the document]
Comment (y-yao) [reply src/chem/integrals.h]: size of these vectors = n_orbs.

**Pull request merged**